api/api_full.go

```diff
@@ -981,7 +980,7 @@ type F3ParticipationLease struct {
 	// Network is the name of the network this lease belongs to.
 	Network gpbft.NetworkName
 	// Issuer is the identity of the node that issued the lease.
-	Issuer peer.ID
+	Issuer []byte
```

Comment:
Can we use a custom type alias of PeerID that gets encoded as a string (base58 encoded) in json, but otherwise gets encoded as bytes? Not super critical, but it's nice being able to read it as a string.

Alternatively, encode it as a base58 string always?

See the OpenRPC example json for why I'm making this comment.

Reply:
Going with base58 encoding everywhere 👍